client/components/App.jsx

```diff
@@ -12,7 +12,7 @@ class App extends React.Component {
       </div>
     )
   }
-  
+
```

Comment:
Not going to split hairs but if you wanted some practice fixing a pull request I don't think any changes to App.jsx need to be a part of this pull request.

Reply:
@cchauche Let me try to fix this. I certainly didn't mean to include this change in my update.